puppet/modules/profiles/manifests/redmine.pp

```diff
@@ -4,12 +4,48 @@
   # Redmine is already included via ENC
   # TODO include redmine
   $backup_path = '/var/lib/redmine'
+  $backup_db_path = "${backup_path}/db"
 
   include profiles::backup::sender
+  include postgresql::server
+  include redmine
+
+  postgresql::server::role { 'restic':
+    password_hash => postgresql::postgresql_password('restic', 'mypasswd'),
```

Comment:
I wonder if it even needs a password. It's probably going to use ident auth.

Reply:
it does use ident, yeah, can be dropped